src/System.Memory/src/System/SpanHelpers.byte.cs

```diff
@@ -105,72 +105,67 @@ public static bool SequenceEqual(ref byte first, ref byte second, int length)
             int index = 0;
```

Comment:
As the param is by ref `ref byte first` is it safe to use `AsPointer`?
e.g. `Unsafe.AsPointer(ref first);`